mono/sgen/sgen-los.c

```diff
@@ -716,9 +716,10 @@ LOSObject*
 sgen_los_header_for_object (GCObject *data)
 {
 #if _MSC_VER
```

Comment:
Upon further inspection.
The msvc code should be the only code.
The other thing should be offsetof from stddef.h.